README.md

```diff
@@ -67,9 +67,7 @@ To contribute to the development of xDSL follow the subsequent steps.
 ```bash
 git clone https://github.com/xdslproject/xdsl.git
 cd xdsl
-pip install --editable .
-# Optional installation of extra requirements
-pip install --requirement requirements-optional.txt
+pip install --editable ".[extras]"
```

Comment:
`pip install --editable `.[extras]` without the `""` never worked for me, maybe it's a zsh thing, but it would try to expand it and fail. The quotes make it work on my machine